packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -485,14 +478,6 @@ export const initializeChugSplash = async (
   }
   logger?.info('[ChugSplash]: finished assigning proposer roles')
 
-  logger?.info('[ChugSplash]: assigning caller roles...')
-  for (const caller of callers) {
-    if ((await ManagedService.hasRole(CALLER_ROLE, caller)) === false) {
-      await ManagedService.connect(signer).grantRole(CALLER_ROLE, caller)
```

Comment:
I think we should change this to:
```
    await (
      await ManagedService.connect(signer).grantRole(CALLER_ROLE, caller, await getGasPriceOverrides(provider))
    ).wait()
```

Adding the extra `await (...).wait()` and using `getGasPriceOverrides` will reduce the likelihood that these transactions fail when we deploy ChugSplash on live networks, particularly Optimism.

Fyi, it seems like these two things are missing from other transactions in this script. They're only necessary on state-changing transactions (i.e. not view functions)